<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreChanger</class>
 <widget class="QMainWindow" name="ScoreChanger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScoreChanger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="matchSetings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Match Settings</string>
    </property>
    <widget class="QLineEdit" name="nameBLU">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="BLU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>BLU</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameRED">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="RED">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RED</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pointBLU">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="pointRED">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboMap">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_badlands</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_granary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_gullywash_final1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_snakewater_final1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_process_final</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>koth_pro_viaduct_rc4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>koth_ashville_rc1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>koth_coalplant_b7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>koth_ultiduo_r_b7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pl_swiftwater_ugc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ctf_turbine_pro_b1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_gravelpit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp_steel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>koth_lakeside_final</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pl_badwater</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pl_upward</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ctf_ballin_sky</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mge_training_v8_beta4a</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mge_training_v8_beta4b</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ultiduo_baloo</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="map">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Map</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetScore">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Score</string>
     </property>
    </widget>
    <widget class="QPushButton" name="swap">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endMatch">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>End Match</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Stream Settings</string>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="titleEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>551</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shedule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Shedule</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="statsEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stats">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stats Link</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="narratorEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="narrator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Narrator</string>
     </property>
    </widget>
    <widget class="QLabel" name="commentator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Commentator</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="commentatorEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="camera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="act">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>240</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ScoreChanger</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPath"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPath">
   <property name="text">
    <string>Path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
